02-household-queries/chainlit-household-bot.py

```diff
@@ -25,6 +39,7 @@ async def init_chat():
                 value="chooseBetter",
                 label="Demo choosing better response",
             ),
+            cl.Action(name="uploadQueryAct", value="upload_query", label="Upload query")
```

Comment:
I don't understand what 'query' needs to be uploaded. Looking at the function, this action is more than just uploading. This action embeds the chosen file into the vector DB. Rename these strings to be more intuitive.